backend/tracim_backend/lib/utils/authorization.py

```diff
@@ -133,18 +133,18 @@ def wrapper(self, context, request: 'TracimRequest') -> typing.Callable:
     return decorator
 
 
-def require_content_types(content_types: typing.List['NewContentType']) -> typing.Callable:  # nopep8
+def require_content_types(content_types: typing.List['ContentType']) -> typing.Callable:  # nopep8
     """
     Restricts access to specific file type or raise an exception.
     Check role for candidate_workspace.
-    :param content_types: list of NewContentType object
+    :param content_types: list of ContentType object
     :return: decorator
     """
     def decorator(func: typing.Callable) -> typing.Callable:
         @functools.wraps(func)
         def wrapper(self, context, request: 'TracimRequest') -> typing.Callable:
             content = request.current_content
-            current_content_type_slug = ContentType(content.type).slug
+            current_content_type_slug = CONTENT_TYPES.get_one_by_slug(content.type).slug
```

Comment:
Why get object by slug to get it's same slug ?

Reply:
Because that allowed to :  
- Check if content slug is correct 
- Support legacy_slug: get_one_by_slug will work will return 'html-document' for 'page' for example.

Reply:
okay